lib/shelley/src/Cardano/Wallet/Shelley/Network/Blockfrost.hs

```diff
@@ -326,6 +337,38 @@ withNetworkLayer tr network np project k = do
         runBFM bfConfig $ liftEither . fromBlockfrostPP networkId
             =<< BF.getLatestEpochProtocolParams
 
+    currentSlottingParameters :: BF.ClientConfig -> IO SlottingParameters
+    currentSlottingParameters bfConfig = runBFM bfConfig do
+        BF.EpochInfo{_epochInfoEpoch} <- BF.getLatestEpoch
+        epochNo <- fromBlockfrostM _epochInfoEpoch
+        let EraParams{..} = currentEraParams epochNo
+        epochLen <- unEpochSize eraEpochSize <?#> "EpochSize"
+        securityParameter <- case eraSafeZone of
+            StandardSafeZone wo -> Quantity <$> wo <?#> "StandardSafeZone"
+            UnsafeIndefiniteSafeZone -> error "UnsafeIndefiniteSafeZone"
+        pure SlottingParameters
+            { getSlotLength = SlotLength $ ST.getSlotLength eraSlotLength
+            , getEpochLength = EpochLength epochLen
+            , getActiveSlotCoefficient = ActiveSlotCoefficient 0.05
+            -- TODO: what's better than hardcoding it          ^^^^ ?
+            , getSecurityParameter = securityParameter
+            }
+      where
+        currentEraParams :: EpochNo -> EraParams
+        currentEraParams (EpochNo epoch) =
+            eraParams . findByEpoch . getSummary $
+                Fixture.networkSummary networkId
+          where
+            findByEpoch :: UC.NonEmpty xs EraSummary -> EraSummary
```

Comment:
(This function seems to be independently useful. Perhaps move it to the top level by turning `epoch` into a parameter?)